scripts/train/train.py

```diff
@@ -259,11 +277,16 @@ def main(cfg: DictConfig):
     ]
 
     # Callbacks
-    callbacks = [
+    callbacks: List[Callback] = [
         build_callback(name, callback_cfg)
         for name, callback_cfg in (cfg.get('callbacks') or {}).items()
     ]
 
+    if model_gauntlet_callback is not None:
+        callbacks.append(model_gauntlet_callback)
+        if not any(isinstance(l, InMemoryLogger) for l in loggers):
+            loggers.append(build_logger('in_memory_logger', {}))
+
```

Comment:
we should never silently add an unwanted callback. suggest we find a different way to pass data, rather than rely on a logger destination.

Reply:
I thiiink `state.eval_metric_values` should work here